homeassistant/components/systemmonitor/sensor.py

```diff
@@ -52,71 +57,226 @@
+        mandatory=True,
     ),
-    "process": ("Process", " ", CPU_ICON, None, True),
-    "processor_use": ("Processor use (percent)", PERCENTAGE, CPU_ICON, None, False),
-    "processor_temperature": (
-        "Processor temperature",
-        TEMP_CELSIUS,
-        None,
-        DEVICE_CLASS_TEMPERATURE,
-        False,
+    SystemMonitorSensorEntityDescription(
+        key="ipv6_address",
+        name="IPv6 address",
+        unit_of_measurement=None,
+        icon="mdi:server-network",
+        device_class=None,
+        state_class=None,
+        mandatory=True,
     ),
-    "swap_free": ("Swap free", DATA_MEBIBYTES, "mdi:harddisk", None, False),
-    "swap_use": ("Swap use", DATA_MEBIBYTES, "mdi:harddisk", None, False),
-    "swap_use_percent": ("Swap use (percent)", PERCENTAGE, "mdi:harddisk", None, False),
-}
+    SystemMonitorSensorEntityDescription(
+        key="last_boot",
+        name="Last boot",
+        unit_of_measurement=None,
+        icon=None,
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        state_class=None,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="load_15m",
+        name="Load (15m)",
+        unit_of_measurement=None,
+        icon=CPU_ICON,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="load_1m",
+        name="Load (1m)",
+        unit_of_measurement=None,
+        icon=CPU_ICON,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="load_5m",
+        name="Load (5m)",
+        unit_of_measurement=None,
+        icon=CPU_ICON,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="memory_free",
+        name="Memory free",
+        unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:memory",
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="memory_use",
+        name="Memory use",
+        unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:memory",
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="memory_use_percent",
+        name="Memory use (percent)",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:memory",
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="network_in",
+        name="Network in",
+        unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:server-network",
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Isn't this sensor going up all the time? (As in, summing data usage)
Should it be STATE_CLASS_TOTAL_INCREASING?